### ISSUE TITLE
Debugger completely broken on Mac

### ISSUE THREAD

**Issue by pesfandiar** [4ian/GDevelop#4618]
## Describe the bug
The debugger tab does not connect to the preview window, and it renders the debugger unfunctional.

## To Reproduce
1. Open a game
2. Using the debugger icon at the top, open "Start preview with Debugger and Performance Profiler"
3. See that a preview window opens correctly
4. See that the debugger tab say "No preview running. ..."
5. Start preview using the icon at the top
6. See that the debugger tab still cannot connect to the preview window

* Please include a link to a game if possible!  
Not published.

* If applicable, add screenshots to help explain your problem.

## Other details
* Include any OS/browser version/smartphone that you're using
macOS 12.5

* Which version of GDevelop are you using? The desktop app or the web-app?
5.1.151 (editor full version: 5.1.151-57ce8dcbbfa08eca1588b132aa98a44d2e59489e, core version: 4.0.99-0-release)

* Add any other context about the problem here.
It's my first time learning and using GDevelop as a hobby. It's a great tool. I tried to use debugger for fixing a bug, but it doesn't seem to work. Thanks!

**Comment by pesfandiar** [comment]
@4ian congratulations on making such a great tool. I'm currently learning it and hoping to play around with it with my child too. 

I didn't see this problem reported elsewhere, so I filed an issue. Let me know if I need to provide more info. Thanks!

**Comment by 4ian** [comment]
Thanks for reporting this! Sounds weird that it does not work - I just tested and it works on my mac (but I'm on a fairly old macOS version).
I'll ask @ClementPasteau and @AlexandreSi to test, they are also on a fairly recent macOS version.

Any chance you could run GDevelop from a terminal? Like this:
`/Applications/GDevelop\ 5.app/Contents/MacOS/GDevelop\ 5`

and paste here the content of the terminal after trying to run the debugger. For example, for me it gives:

```
/Applications/GDevelop\ 5.app/Contents/MacOS/GDevelop\ 5
13:35:15.844 › GDevelop Electron app starting...
13:35:28.942 › Starting check for updates (with auto-download if any)
13:35:28.944 › Checking for update
13:35:28.945 › { message: 'Checking for update...', status: 'checking-for-update' }
13:35:30.884 › Update for version 5.1.151 is not available (latest version: 5.1.151, downgrade is disallowed).
13:35:30.885 › { message: 'Update not available.', status: 'update-not-available' }
13:35:30.887 › checkForUpdatesAndNotify called, downloadPromise is null
13:35:45.850 › Received event to start debugger server with options= null
13:35:45.853 › Debugger listening for connections.
13:35:46.731 › Debugger connection #0 opened.
13:35:46.881 › Debugger connection #0 received message.
13:35:47.495 › Debugger connection #0 received message.
13:35:47.564 › Debugger connection #0 received message.
13:35:47.579 › Debugger connection #0 received message.
13:35:47.580 › Debugger connection #0 received message.
```

**Comment by pesfandiar** [comment]
I should've tried this before, but it seems like merely closing and opening the app again resolved it. Closing it. Thanks
